tests/test_item.rs

```diff
@@ -514,3 +514,43 @@ fn test_type() {
         })
     );
 }
+
+#[test]
+fn test_impl() {
+    let builder = AstBuilder::new();
+    let impl_ = builder.item().impl_()
+        .trait_().id("ser::Serialize")
```

Comment:
Sure I'll do that to promote good behavior/proper usage.  Technically the AST doesn't care though.